Source/WebKit/UIProcess/API/C/WKPreferences.cpp

```diff
@@ -1557,16 +1557,6 @@ bool WKPreferencesGetIntersectionObserverEnabled(WKPreferencesRef preferencesRef
     return toImpl(preferencesRef)->intersectionObserverEnabled();
 }
 
-void WKPreferencesSetMenuItemElementEnabled(WKPreferencesRef preferencesRef, bool flag)
```

Comment:
It might be a good idea to keep these two SPI functions around as an empty stub, only for bincompat.